shared/middleware/helpers/storage_apps.rb

```diff
@@ -13,10 +13,12 @@ def initialize(storage_id)
   end
 
   def create(value, ip_address)
+    timestamp = Time.now
```

Comment:
It looks like there's some back and forth on which is better.  I'll stick with `DateTime` for now to avoid too much churn in this PR.  If we decide to change to `Time` later it should be easy to update everywhere.

Links:
http://stackoverflow.com/questions/5941613/are-the-date-time-and-datetime-classes-necessary
https://gist.github.com/pixeltrix/e2298822dd89d854444b